src/air/routing.py

```diff
@@ -161,10 +162,21 @@ def get_user(user_id: int):
 
             # The .url() method is created by this helper
             url = get_user.url(user_id=123)  # Returns: "/users/123"
+            url_with_query = get_user.url(user_id=123, query_params={"page": 2})
+                # Returns: "/users/123?page=2"
         """
 
         def helper_function(**params: Any) -> str:
-            return self.url_path_for(name, **params)
+            query_params = params.pop("query_params", None)
```

Comment:
I like the elegance and robust nature of how you handle `query_params` in the helper function. 

The only complaint is that `query_params` is that while explicit, it is a long name for the argument. If another name comes up that is as explicit but shorter, maybe we'll add that as well.

No action needed, just commentary.

Reply:
I thought of `parameters` and `criteria`, but wasn't sure enough if it's something that would make sense to others. I made use of `query_params`, simply because it is considered "industry standard". 

Let me know what you think.